services/src/main/java/org/keycloak/protocol/oid4vc/issuance/OID4VCIssuerWellKnownProvider.java

```diff
@@ -192,6 +209,37 @@ public static List<String> getSupportedEncryptionAlgorithms(KeycloakSession sess
         return supportedEncryptionAlgorithms;
     }
 
+
+    /**
+     * Builds JWKS from realm encryption keys with use=enc.
+     */
+    private static JSONWebKeySet buildJwks(KeycloakSession session) {
```

Comment:
It would be good to also have the ability to configure the keys(or key-types) to be included. Practically, only asymmetric keys can be used for the encryption, thus some wallets(for example github.com/eu-digital-identity-wallet/eudi-app-android-wallet-ui) consider symmetric keys as part of the meta-data as invalid and abort the issuance process.